src/integrations/third-party/jetpack.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package Yoast\WP\SEO\Integrations\Third_Party
+ */
+
+namespace Yoast\WP\SEO\Integrations\Third_Party;
+
+use Yoast\WP\SEO\Conditionals\Front_End_Conditional;
+use Yoast\WP\SEO\Conditionals\JetPack_Conditional;
+use Yoast\WP\SEO\Conditionals\OpenGraph_Conditional;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+
+/**
+ * Class Jetpack
+ */
+class Jetpack implements Integration_Interface {
```

Comment:
Could the name of this class lead to problems or confusion in the future? Since `Jetpack` is also a class name used by the Jetpack plugin. Or is the namespace sufficient to fix any issues?

Reply:
This is what namespaces are for. The Jetpack is in the global namespace (since there is no namespace specified there). This interface is in the `Yoast\WP\SEO\Integrations\Third_Party` namespace.